<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_interfere_analysis</class>
 <widget class="QWidget" name="tab_interfere_analysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="C2I">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>主邻小区C2I干扰分析</string>
   </property>
  </widget>
  <widget class="QPushButton" name="triple">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>查询重叠覆盖干扰三元组</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="C2I_path">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="C2I_file">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>……</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="triple_path">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="triple_file">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string> …</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
